floodviz/views.py

```diff
@@ -1,7 +1,79 @@
-from flask import request, render_template, abort
+from flask import request, render_template, abort #, jsonify
+
+## below is for data processing ##
+import json
+import time
+from datetime import datetime
+import requests
+import operator
+# # # # # # # # # # # # # # # # # #
 
 from . import app
 
 @app.route('/home/')
 def home():
-    return render_template('index.html')
\ No newline at end of file
+  	## new data store ##
+	newd = []
+	sites_value_maxs = {}
+	sites = app.config['SITE_IDS'] 
+	start_date = app.config['START_DT']
+	end_date = app.config['END_DT']
+	n_show_series = app.config['N_SERIES']
+	hydro_meta = app.config['HYDRO_META']
+
+	## Set up to retrieve all site ids from url? ##
+	for idx, site in enumerate(sites):
+		url = 'https://nwis.waterservices.usgs.gov/nwis/iv/?site=' + site + '&startDT=' + start_date + '&endDT=' + end_date + '&parameterCD=00060&format=json'
+
+		try:
+			r = requests.get(url)
+		except:
```

Comment:
Was just using that intermediately for overall fault tolerance. But later added the response code if statement and did not remove the try. I must see them too much and glanced right over.